packages/integration-react/api-report.md

```diff
@@ -8,6 +8,11 @@ import { ApiRef } from '@backstage/core';
 import { Config } from '@backstage/config';
 import { ScmIntegrationRegistry } from '@backstage/integration';
 
+// @public (undocumented)
+export const ScmIntegrationIcon: ({ type }: {
```

Comment:
Hm! I wonder what the ideal interface would be. What we have here is one possible input, and that's fine. Wondering if we could also later gear it toward supporting a per-integration `icon` property (set in the integration config settings), so that you can have one icon for your public github integration and one for your ghe for example. Then maybe `host` and/or `url` could be a sensible prop to this component.

Reply:
Hmm, interesting thought. I could add it in a way that we can later add a real implementation. 
I would like to get this into the release tomorrow to avoid doint a breaking change 👍

Reply:
It's approved and ready to go, and I think you can keep this prop while adding another alternative in the future.